### PR TITLE
Task1 Models part 2

### PR DESCRIPTION
**Pull request by Kaid000**
From last PR i didn't resolved  are 2 problems:
For what i would can use files?
How do i can rename News class correct?

### CODE REVIEW

**Review by BlazeOfSoul — COMMENTED** [review]
Why folder "obj" in PR? Delete it from PR and add it to gitignore

File: server/RAFwebsite.Models/Event.cs [hunk 1]
```diff
@@ -0,0 +1,12 @@
+﻿namespace RAFwebsite.Models
+{
+    public class Event : BaseEntity
+    {
+        public int Id { get; set; }
+        public DateTime Datetime { get; set; }
```
[the remark this answers is not in the document]
Comment (Kaid000) [reply server/RAFwebsite.Models/Event.cs:6]: Yes. When the last event was

File: server/RAFwebsite.Models/ContactInfo.cs [hunk 2]
```diff
@@ -0,0 +1,9 @@
+﻿namespace RAFwebsite.Models
+{
+    public class ContactInfo : BaseEntity
+    {
+        public string Text { get; set; }
+        public int Phote { get; set; }
```
[the remark this answers is not in the document]
Comment (BlazeOfSoul) [reply server/RAFwebsite.Models/ContactInfo.cs:6]: string **Phone**

File: server/RAFwebsite.Models/StudentOrganization.cs [hunk 3]
```diff
@@ -0,0 +1,10 @@
+﻿namespace RAFwebsite.Models
+{
+    public class StudentOrganization : BaseEntity
+    {
+            public string Name { get; set; }
+            public string Ruler { get; set; }
```
Comment (BlazeOfSoul) [inline server/RAFwebsite.Models/StudentOrganization.cs:6]: Rename to Leader

**Pull request closed without merging**